flang/lib/Lower/IO.cpp

```diff
@@ -255,7 +255,7 @@ static void genInputRuntimeFunc(Fortran::lower::FirOpBuilder &builder,
   auto inputFunc = getInputRuntimeFunc(builder, argType, intKind);
   if (auto ty = argType.dyn_cast<fir::CplxType>()) {
     var = builder.create<fir::CoordinateOp>(
-        loc, builder.getRefType(ty.getEleTy()), var,
```

Comment:
Why was this wrong ?

Reply:
I changed it to use the same path through the builder to be consistent.